README.md

```diff
@@ -1 +1,61 @@
-# To-Do-List
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do List
+
+>This is a website that allows users to manage the tasks. users can add every task to do and when the task is done users can delete the task
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Linters
+- webpack
+
+## Live Demo
+
+[Live Demo Link]()
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+  - IDE eg (Atom/Visual studio code)
+  - Set-up linter(HTML and CSS)
+  - webpack
+
+### Setup
+  - Clone the github repository and set up linters and webpack
+
```

Comment:
- [ ] For the benefit of non-technical/unfamiliar users kindly include more specific instructions on how to clone the repo and run the app locally.